src/components/CommentList/CommentList.scss

```diff
@@ -1 +1,8 @@
-//
+.CommentList {
+  display: flex;
+  flex-direction: column;
+  gap: 0.7em;
+  background-color: #eee;
+  padding: 1em;
+}
+
```

Comment:
Here you hava a linter error. Dont forget to run lint tests befor pushing your code on github